x-pack/plugins/cases/public/api/index.test.ts

```diff
@@ -47,4 +47,23 @@ describe('api', () => {
       });
     });
   });
+
+  describe('bulkGetCases', () => {
+    const http = httpServiceMock.createStartContract({ basePath: '' });
+    http.post.mockResolvedValue({ cases: allCasesSnake, errors: [] });
+
+    it('should return the correct response', async () => {
+      expect(await bulkGetCases({ http, params: { ids: ['test'], fields: ['title'] } })).toEqual({
```

Comment:
Maybe a nit but considering this is `bulkGet` could the tests ask for more than one `id`?

Reply:
I omitted it because there is no logic regarding the number of items in the `ids` list. Do you want to avoid a bug introduced in the future?